src/core/namespace.c

```diff
@@ -113,6 +113,9 @@ typedef struct MountEntry {
         LIST_HEAD(MountOptions, image_options_const);
         char **overlay_layers;
         VeritySettings verity;
+        bool idmapped;
+        uid_t uid;
```

Comment:
maybe rename this `idmap_uid`, because that's really what is for.

(many fs have uid= switches, which this is not about, and we shouldn't create confusion on this)